src/TestcafeGherkinBootstrapper.js

```diff
@@ -146,6 +145,15 @@ module.exports = class TestcafeGherkinBootstrapper extends TestcafeBootstrapper
     );
   }
 
+  _shouldRunStep(stepDefinition, step) {
```

Comment:
Now that I think of it, it's not good to not return anything if the step definition's pattern is neither a string nor a RegExp. I think in that case, we should either an error.

Reply:
Do you reckon this makes sense? `throw new Error(`Step implementation is not valid for: ${step.text}`);`

Reply:
Yeah, you are right. It would certainly not point me into the right direction.

I would sugget an error message like "Step implementation invalid. Has to be a string or RegExp. Received...".